src/js/views/homeView.js

```diff
@@ -143,7 +141,30 @@ export const expand = () => {
     document.querySelector('.more-less').textContent = "Show less profile information";
 };
 
-
+export const setEditUI = (state) => {
+    state.home.laputa = document.querySelector(".laputa");
+    state.home.finishBtn = document.querySelector("#finish-btn");
+    state.home.about = document.querySelector('#laputa-about-area');

```

Comment:
Although the html tag's id is somewhat descriptive (I'd prefer `update`/`edit` over laputa), the `state`'s key is not. Can we make it more specific?

Reply:
We will change that ASAP